app/search/page.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
```

Comment:
1. Please use only 1 page for the current task
2. You can set URL without changing the route 
3. Done make API request directly in the component because they will fire on each render, so put it inside useEffect with async IIFE

Reply:
Ok, in this case getNews fires in 'use client' component and it is not allowed to use .env variables without NEXT_PUBLIC prefix. Is it OK to store API_KEY like that?

Reply:
Yes, it is